specify/specify7
# Filter out unmapped field in formatter editor

Fixes #4747

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Go to record formatters (through app resources or schema config)
- Click on a table
- Click on a formatter
- Add a field
- Do not map it
- Close the record formatter editor overlay 
- You are back to the table list 
- Click save 
- Reopen you record formatter 

- [ ] verify that the unmapped fields have been deleted
